src/test/java/parser/JsonParserTest.java

```diff
@@ -0,0 +1,65 @@
+package parser;
+
+import org.testng.annotations.*;
+import org.testng.asserts.SoftAssert;
+import shop.Cart;
+
+import java.io.*;
+
+import static org.testng.Assert.*;
+
+
+class JsonParserTest {
+
+    private JsonParser parser;
+    public static final String CART_NAME = "andrew-cart";
+
+    @BeforeMethod
```

Comment:
remove this annotation from data provider method. '@BeforeMethod' annotation is for `init()` method